README.md

```diff
@@ -31,10 +31,9 @@ Note: Support for the ISO is community based. Before filing a new bug or feature
 
 
 ## Getting Help

```

Comment:
Do we want to target `2.0`  or `main` ?

Reply:
I assume main but I'll let you guys decide :)